generic_config_updater/gu_common.py

```diff
@@ -212,6 +214,41 @@ def _invoke_validating_function(cmd, jsonpatch_element):
                 if not _invoke_validating_function(function, element):
                     raise IllegalPatchOperationError("Modification of {} table is illegal- validating function {} returned False".format(table, function))
 
+    def illegal_dataacl_check(self, old_config, upd_config):
+        # Block data ACL changes when patch includes:
+        # 1. table "type" being replaced
+        # 2. rule update on tables with table "type" replaced
+        # This will cause rase condition when swss consume the change
+        # and make acl rule inactive
+        old_acltable = old_config.get("ACL_TABLE", {})
+        upd_acltable = upd_config.get("ACL_TABLE", {})
```

Comment:
`upd` for `updated` can easily confuse people. Suggest always spelling out the full word. With modern formatting tools it is better to have a long variable name than having a variable name whose meaning is unclear to readers.

https://semirise.com/variable-naming-conventions-and-best-practices

Reply:
Will keep upd as it was applied all over the code in GCU.